app/experimenter/visualization/api/v1/views.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+from django.core.files.storage import default_storage
+from django.http import HttpResponse
+from django.views import View
+
+from experimenter.experiments.models import Experiment
+
+
+class VisualizationView(View):
+    def load_data_from_gcs(self, filename):
+        return (
+            json.loads(default_storage.open(filename).read().decode("utf8"))
+            if default_storage.exists(filename)
+            else []
+        )
```

Comment:
I see below you're just using a truthy test to check for the case that nothing gets read, but you could also just return `None` here so there's no confusion between the read being successful but empty, vs not being able to find the file?